app/src/views/Quiz.jsx

```diff
@@ -0,0 +1,46 @@
+import React, { useState, useEffect } from 'react';
+import { Text, View } from 'react-native';
+import Swiper from 'react-native-swiper';
```

Comment:
Do we want to use Swiper stuff for the quiz or do we want to show them only one at a time?